src/components/Task/Modal/Form/Complete/TaskModalCompleteForm.tsx

```diff
@@ -222,6 +222,10 @@ const TaskModalCompleteForm = ({
       openTaskModal({
         view: TaskModalEnum.Add,
         defaultValues: {
+          subject:
+            attributes.nextAction === task.activityType
```

Comment:
Can you add a comment here that helps the dev understand why this is here? 

I'm thinking of devs who come to this project, might not understand this logic straight away.